src/arch/wasm/CodeGen.zig

```diff
@@ -6446,63 +6446,89 @@ fn airDivFloor(func: *CodeGen, inst: Air.Inst.Index) InnerError!void {
     const rhs = try func.resolveInst(bin_op.rhs);
 
     if (ty.isUnsignedInt(mod)) {
-        const result = try (try func.binOp(lhs, rhs, ty, .div)).toLocal(func, ty);
-        return func.finishAir(inst, result, &.{ bin_op.lhs, bin_op.rhs });
+        _ = try func.binOp(lhs, rhs, ty, .div);
```

Comment:
What's the reasoning behind this change? If anything, removing the early return makes it harder to follow the code and easier to mistake where the result is being returned, yet removing the early return does not contribute to any improvements in generated code.

Reply:
All three cases (unsigned, signed, float) end by allocating a local to store the result and then calling `func.finishAir`, but in the unsigned case this code was duplicated, with an early return to prevent this final logic from running twice. The reasoning behind the change was to remove the duplication.

Reply:
Duplication isn't always a bad thing. Especially, in my opinion, when it reduces mistakes. Anyway, was just curious about the reasoning.